### PR TITLE
Alearts

### PR DESCRIPTION
**Pull request by bossmonkey98**
Alerts Updated

### CODE REVIEW

File: pages/alerts/alerts.css [hunk 1]
```diff
@@ -20,13 +20,18 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
     font-size:small;
     padding: 2px;
     margin-right:10px;
     border: 1px solid green;
     border-radius: 50%;
     color: aliceblue;
     font-weight: 500;
```
Comment (arjun-computer-geek) [inline pages/alerts/alerts.css:32]: Use color variable here

**Review by arjun-computer-geek — APPROVED** [review]
Else are good.

File: pages/alerts/alerts.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    background-color: var(--primary);
+    border: none;
+    border-radius: 5px;
+}
+.btn:hover{
+    background-color: var(--btn-bg);
+    border: 1px solid var(--primary);
+}
+
+.primary-alert,.secondary-alert,.danger-alert,.warning-alert,.success-alert{
```
Comment (t007rushi) [inline pages/alerts/alerts.css:79]: create a common class for this

**Review by t007rushi — APPROVED** [review]
Go ahead with suggestions 
all things good to me

File: pages/alerts/alerts.css (hunk 2 above)
Comment (bossmonkey98) [reply pages/alerts/alerts.css:79]: ok

File: pages/alerts/alerts.css (hunk 1 above)
Comment (bossmonkey98) [reply pages/alerts/alerts.css:32]: ok

**Pull request merged**